src/main/resources/static/js/login.js

```diff
@@ -34,6 +34,7 @@ var frontendServiceUrl = $('#frontendServiceUrl').text();
 			},
 			error : function (request, textStatus, errorThrown) {
 				$.jGrowl("Bad credentials", { sticky : false, theme : 'Error' });
+				errorsStore.add("Bad credentials");
```

Comment:
jGrowl and errorStore can be moved out an function e.g. logMessage(message) , and call this function where we do jGrowl.

Reply:
Do you mean create a method with jGrowl and errorsStore where we set message?
Something like
logMessage(message){
jGrowl(message);
errorsStore(message);
}

Reply:
yes